sources/api/prairiedog/Cargo.toml

```diff
@@ -20,7 +20,7 @@ signpost = { path = "../../updater/signpost", version = "0.1" }
 simplelog = "0.12"
 snafu = "0.7"
 serde_json = "1"
-tokio = { version = "~1.32", default-features = false, features = ["macros", "rt-multi-thread"] } # LTS
```

Comment:
One of the primary purposes of this work is to remove the dependency on `models`. Can you please remove it?